### DISCUSSION TITLE
What is a "field array"?

### DISCUSSION THREAD

**Discussion by ersinakinci** [react-hook-form/react-hook-form#2952] [Q&A]
The documentation talks about a `useFieldArray` hook and there's a section in the advanced usage that describes how to use it, but nowhere actually states what a "field array" is or why one should use it.

I'm asking because I think that field arrays might (?) solve a problem I'm currently hacking my way around, but I'm not sure because I don't actually know what it is 🙃.

**Comment by bluebill1049** [comment]
honestly, I hate the word field array, it should be called dynamic fields... but this is what they call it in the industry.

**Reply by ersinakinci** [reply]
Ok...but what is it? Can you give a definition? And can that be added to the docs?

**Reply by bluebill1049** [reply]
i can include it (or PR welcome as well), i am probably not the best person to give a definition. if you find some good reference happy to be included in the doc.
